jsx/DataTable.js

```diff
@@ -317,7 +317,7 @@ class DataTable extends Component {
         searchKey = filterData[i].toLowerCase();
         searchString = data ? data.toString().toLowerCase() : '';
 
-        match = (searchString === searchKey);
+        match = (searchString.indexOf(searchKey) > -1);
```

Comment:
Doing this will give a more precise result, and change to Filter.js not needed:
```suggestion
        let searchArray = searchString.split(',');
        match = (searchArray.includes(searchKey));
```